### PR TITLE
add react router

### PR DESCRIPTION
**Pull request by fickryiman**
Adding React Router
- npm install react router
- add home components
- add navbar components
- add calculator components
- add quote components

### CODE REVIEW

File: src/components/Quote.jsx [hunk 1]
```diff
@@ -52,9 +52,9 @@ function Quote() {
       <h2>
         {quote}
         <span className="author">
-          &nbsp;(
+          &nbsp;
+          ~
           { author}
-          )
         </span>
```
Comment (DuaneDave) [inline src/components/Quote.jsx]: please remove the console.log statement in line 38 and perform an action if there is a potential error in order to make your code follow best [practices](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md#:~:text=Follow%20our%20list%20of%20best%20practices%20for%20JavaScript.)

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @fickryiman 👋 , 

Good job so far on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ good looking design


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Quote.jsx (hunk 1 above)
Comment (fickryiman) [reply src/components/Quote.jsx]: remove console.log and add error message if fetch API got an error

**Review by emmiiorji — APPROVED** [review]
Hi  @fickryiman, 

# Status: Approved ✅

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<div align='center'>

</div>

## Good Highlights

- Requested change has been implemented

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by fickryiman** [comment]
Thank you for the review's

**Pull request merged**